ambry-router/src/main/java/com.github.ambry.router/NonBlockingRouterMetrics.java

```diff
@@ -398,6 +407,15 @@ void onDeleteBlobError(Exception e) {
   }
 
   /**
+   * Update appropriate metrics on a {@link GetBlobOperation} with a valid {@link GetBlobRange}.
```

Comment:
can we explicitly say what metric we are updating here since it is just one metric